src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrentInfo.cs

```diff
@@ -20,46 +21,35 @@ public static IEnumerable<object[]> CurrentInfo_CustomCulture_TestData()
         [MemberData(nameof(CurrentInfo_CustomCulture_TestData))]
         public void CurrentInfo_CustomCulture(CultureInfo newCurrentCulture)
         {
-            CultureInfo oldCurrentCulture = CultureInfo.CurrentCulture;
-            try
+            RemoteInvoke((cultureCode) =>
             {
-                CultureInfo.CurrentCulture = newCurrentCulture;
-                Assert.Same(newCurrentCulture.NumberFormat, NumberFormatInfo.CurrentInfo);
-            }
-            finally
-            {
-                CultureInfo.CurrentCulture = oldCurrentCulture;
-            }
+                CultureInfo newCulture = new CultureInfo(cultureCode);
+                CultureInfo.CurrentCulture = newCulture;
+                Assert.Same(newCulture.NumberFormat, NumberFormatInfo.CurrentInfo);
+                return SuccessExitCode;
+            }, newCurrentCulture.TwoLetterISOLanguageName).Dispose();
```

Comment:
> The empty string was being whitespace-parsed away by the RemoteInvoke call. 

Then you can special-case it, instead of an empty string sending some known sentinel that you interpret as the empty string on the other side.